daemon/gamemode-sched.c

```diff
@@ -74,11 +69,7 @@ void game_mode_apply_renice(const GameModeContext *self, const pid_t client)
 	long int renice = 0;
 	config_get_renice_value(config, &renice);
 	if ((renice < 1) || (renice > 20)) {
-		LOG_ONCE(ERROR,
-		         "Invalid renice value '%ld' reset to default: %d.\n",
-		         renice,
-		         -NICE_DEFAULT_PRIORITY);
-		renice = NICE_DEFAULT_PRIORITY;
+		LOG_ONCE(ERROR, "Invalid renice value '%ld' is invalid, will not renice.\n", renice);
```

Comment:
"Invalid ... is invalid"

Reply:
Haha, touche, I'll fix it up!

Reply:
66bd6a3 fixes this horrible error. Also makes it actually bail out, as it really should.